### PR TITLE
Migrate from `master` to `main` branch

### PR DESCRIPTION
**Pull request by michalinacienciala**
Use of the `master` and `slave` terms in the computer industry is
widespread, but has been recently discouraged, as it was brought to
attention that the terms may be considered offensive.
GitHub has already moved their default repositories from the `master`
to `main` in many places. We're planning to do the same for our
repositories, this however requires some changes in the code.
After the proposed changes get merged to the `master` branch, the
repository should be ready for the migration from `master` to `main`.

To migrate from `master` to `main`:
1. Do a local branch rename with `git branch -m master main`
2. Push the renamed branch `git push -u origin main`
3. In the Github repo settings, rename the default branch: Settings ->
    Branches -> Default branch -> (pencil icon) -> (change `master` to
    `main`) -> (approve)

After migration other contributors should update their local repositories:
```
git branch -m master main
git fetch origin
git branch -u origin/main main
git remote set-head origin -a
```

Refs:
https://github.com/keep-network/keep-core/pull/2521
https://github.com/keep-network/keep-ecdsa/pull/845
https://github.com/keep-network/tbtc-dapp/pull/399
https://github.com/keep-network/tbtc.js/pull/126
https://github.com/keep-network/keep-common/pull/79
https://github.com/keep-network/sortition-pools/pull/113
https://github.com/keep-network/local-setup/pull/97
https://github.com/keep-network/ci/pull/8
https://github.com/keep-network/run-workflow/pull/3
https://github.com/keep-network/notify-workflow-completed/pull/3
https://github.com/keep-network/load-env-variables/pull/2

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -98,7 +98,7 @@ repository. To start a discussion, prefer https://discord.gg/4R6RGFf[Discord]
 over GitHub issues.
 
 *Read the
-https://github.com/keep-network/tbtc/blob/master/CONTRIBUTING.md[Contributing
+https://github.com/keep-network/tbtc/blob/main/CONTRIBUTING.md[Contributing
```
Comment (mhluongo) [inline README.adoc:101]: Could we make this a relative link rather than referencing a branch?

File: docs/introduction-to-tbtc.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ The redemption flow is as follows:
 
 The "happy paths" have been covered, but there hasn't been discussion of when *things fall apart*. 
 
-To disincentivise signers from stealing bitcoin, deposits are overcollateralised with ETH. The collateral is priced using an on-chain [ETH:BTC price feed](https://github.com/keep-network/tbtc/blob/master/solidity/contracts/price-feed/SatWeiPriceFeed.sol), which works using a price-feed operated by MakerDAO. This collateral also ensures  guarantees of signer availability within the protocol.
+To disincentivise signers from stealing bitcoin, deposits are overcollateralised with ETH. The collateral is priced using an on-chain [ETH:BTC price feed](https://github.com/keep-network/tbtc/blob/main/solidity/contracts/price-feed/SatWeiPriceFeed.sol), which works using a price-feed operated by MakerDAO. This collateral also ensures  guarantees of signer availability within the protocol.
```
Comment (mhluongo) [inline docs/introduction-to-tbtc.md:90]: Ditto, how about a relative link?

**Review by mhluongo — APPROVED** [review]
Questions about relative links in the docs versus hardcoding branches... but otherwise LGTM and could merge without it.

File: README.adoc (hunk 1 above)
Comment (michalinacienciala) [reply README.adoc]: Done, see c5e10dd4c35b34d8d0df8f5572e4138567fff802.

File: docs/introduction-to-tbtc.md (hunk 2 above)
Comment (michalinacienciala) [reply docs/introduction-to-tbtc.md]: Done, see c5e10dd4c35b34d8d0df8f5572e4138567fff802.

**Pull request merged**